app.py

```diff
@@ -32,55 +38,64 @@
 
 # Configuration for JWT and session settings
 # Fetch secret key or generate a new one if not available
-SECRET_KEY = os.getenv('SECRET_KEY', str(uuid.uuid4()))
+SECRET_KEY = os.getenv("SECRET_KEY", str(uuid.uuid4()))
```

Comment:
not sure I agree with arbitrary changes from single to double quotes. was that suggested by the linting tool? can you explain.

because pep8 says this:

https://peps.python.org/pep-0008/#string-quotes

In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

Reply:
this was done by black formatter.

Reply:
ah ok, I bow to black linter rules then